src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_quantization.cpp

```diff
@@ -511,6 +511,9 @@ static void optimize_weights_decompression_parameters(fully_connected_node& fc_n
     auto need_reorder = [&](size_t dep_id) {
         auto dep_layout = fc_node.get_input_layout(dep_id);
         auto dep_pshape = dep_layout.get_partial_shape();
+        if (dep_pshape.size() == 0)
+            return false;
```

Comment:
could you explain when this can happen? how can the size be 0?

Reply:
ConvertU4WeightsZeroPointToScalar pass does it. I will add explanation about it.